src/Nuget/BinSkim.nuspec

```diff
@@ -1,79 +1,27 @@
 <?xml version="1.0"?>
 <package >
   <metadata>
-    <id>$id$</id>
-    <version>$major$.$minor$.$patch$$prerelease$</version>
-    <title>Microsoft BinSkim Binary Analysis Tool (x86/x64)</title>
+    <id>Microsoft.CodeAnalysis.BinSkim</id>
+    <version>$version$</version>
+    <title>Microsoft BinSkim Binary Analysis Tool</title>
     <authors>@michaelcfanning,@nguerrera,@billyoneal</authors>
     <owners>@michaelcfanning,@nguerrera</owners>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <description>A correctness and security checker for Windows portable executables</description>
-    <releaseNotes>Version $major$.$minor$.$patch$$prerelease$ of BinSkim</releaseNotes>
+    <releaseNotes>Version $version$ of BinSkim</releaseNotes>
     <copyright>Copyright Microsoft 2015</copyright>    
     <licenseUrl>https://github.com/microsoft/binskim/blob/master/LICENSE</licenseUrl>
     <projectUrl>https://github.com/microsoft/binskim</projectUrl>
     <tags>Binary,Analysis,binskim,binscope</tags>
   </metadata>
   <files>
-    <file src="..\..\src\ReleaseHistory.md" target="tools\ReleaseHistory.md" />
+	<!-- NOTE: the exclude attribute must be on a single line -->
+    <file src="bld\bin\AnyCPU_$configuration$\Publish\**\*.*" target="tools"/>

```

Comment:
There is a lot in the output that we don't want to include in the NuGet packages, e.g. unit test DLLs. That stuff should be excluded.

Reply:
The "Publish" action should exclude that stuff automatically, since it's only called on the Driver project.